htdocs/install/mysql/tables/llx_opensurvey_user_studs-opensurvey.sql

```diff
@@ -19,6 +19,6 @@ CREATE TABLE llx_opensurvey_user_studs (
     id_users INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
     nom VARCHAR(64) NOT NULL,
     id_sondage VARCHAR(16) NOT NULL,
-    reponses VARCHAR(100) NOT NULL,		-- Not used for 'F' surveys
+    reponses VARCHAR(200) NOT NULL,		-- Not used for 'F' surveys
```

Comment:
can you add a request to update in migration 16 -> 17

Reply:
yes well noted , i will fix it